docs/reference/requirement-sets/excel-api-1-4-requirement-set.md

```diff
@@ -35,7 +35,7 @@ APIs include `getItem()` to get setting entry via the key and `add()` to add the
 * Add a table column to the end of the table.
 * Add multiple rows to a table at a time.
 * `range.getColumnsAfter(count: number)` and `range.getColumnsBefore(count: number)` to get a certain number of columns to the right/left of the current Range object.
-* The [get item or null object function](../../excel/excel-add-ins-advanced-concepts.md#ornullobject-methods): This functionality allows getting object using a key. If the object does not exist, the returned object's `isNullObject` property will be true. This allows developers to check if an object exists or not without having to handle it through exception handling. The `*OrNullObject` method is available on most collection objects.
+* The [\*OrNullObject methods and properties](../../develop/host-specific-api-model.md##ornullobject-methods-and-properties): This functionality allows getting an object using a key. If the object does not exist, the returned object's `isNullObject` property will be true. This allows developers to check if an object exists without having to handle it through exception handling. An `*OrNullObject` method is available on most collection objects.
```

Comment:
Does the anchor need two hashtags?
```suggestion
* The [\*OrNullObject methods and properties](../../develop/host-specific-api-model.md#ornullobject-methods-and-properties): This functionality allows getting an object using a key. If the object does not exist, the returned object's `isNullObject` property will be true. This allows developers to check if an object exists without having to handle it through exception handling. An `*OrNullObject` method is available on most collection objects.
```

Reply:
No, I just copy and pasted the same error everywhere.